File: iniciante/javascript/1061.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

function relogio(a, b, c, d, e, f, g, h){
    let diaInicio = a
    let horaInicio = b
    let minInicio = c
    let segundoInicio = d
    let diaFinal = e
    let horaFinal = f
    let minFinal = g
    let segundoFinal = h
    let countDia = 0
    let countHora = 0
    let countMinuto = 0
    let countSegundo = 0

    //Segundo
    while(true){
        segundoInicio++
        countSegundo++

        if(countSegundo == 60){
            countMinuto++
            countSegundo = 0
        }

        if(segundoInicio == segundoFinal){
            break
        }

        if(segundoInicio == 59){
            segundoInicio = -1
            minInicio++
        }

    }

    //Minuto
    while(true){

        if(minInicio == minFinal){
            break
        }

        minInicio++
        countMinuto++

        if(countMinuto == 60){
            countHora++
            countMinuto = 0
        }

        if(minInicio == 59){
            minInicio = -1
            horaInicio++
        }

    }
    
    //Hora
    while(true){
        
        if(horaInicio == horaFinal){
            break
        }
        
        horaInicio++
        countHora++

        if(countHora == 24){
            countDia++
            countHora = 0
        }

        if(horaInicio == 23){
            horaInicio = -1
            diaInicio++
        }

    }
    //Dia
    while(true){
        if(diaInicio == diaFinal){
            break
        }

        diaInicio++
        countDia++ 

    }

   
    return `${countDia} dia(s)\n${countHora} hora(s)\n${countMinuto} minuto(s)\n${countSegundo} segundo(s)`
}


let entrada1 = lines.shift().split(' ')
let entrada2 = lines.shift().split(' : ')
let entrada3 = lines.shift().split(' ')
let entrada4 = lines.shift().split(' : ')

let a = parseInt(entrada1[1].replace('\r',''))
let b = parseInt(entrada2[0])
let c = parseInt(entrada2[1])
let d = parseInt(entrada2[2].replace('\r',''))
let e = parseInt(entrada3[1].replace('\r',''))
let f = parseInt(entrada4[0])
let g = parseInt(entrada4[1])
let h = parseInt(entrada4[2].replace('\r',''))

console.log(relogio(a, b, c, d, e, f, g, h))

File: iniciante/python/1061.py
# -*- coding: utf-8 -*-

entrada1 = input()
entrada2 = input()
entrada3 = input()
entrada4 = input()

dia_inicio = entrada1.split(' ')
dia_inicio = int(dia_inicio[1])
dia_final = entrada3.split(' ')
dia_final = int(dia_final[1])
hora_inicio = int(entrada2[0:2])
hora_final = int(entrada4[0:2])
minuto_inicio = int(entrada2[5:7])
minuto_final = int(entrada4[5:7])
segundo_inicio = int(entrada2[-2:])
segundo_final = int(entrada4[-2:])

dia = dia_final - dia_inicio
hora = hora_final - hora_inicio
minuto = minuto_final - minuto_inicio
segundo = segundo_final - segundo_inicio

if segundo < 0:
    segundo = 60 + segundo
    minuto -= 1
if minuto < 0:
    minuto = 60 + minuto
    hora -= 1
if hora < 0:
    hora = 24 + hora
    dia -= 1


print(f'{dia} dia(s)')
print(f'{hora} hora(s)')
print(f'{minuto} minuto(s)')
print(f'{segundo} segundo(s)')
